control-station/src/App.tsx

```diff
@@ -1,13 +1,27 @@
-import { Dashboard } from "@/views";
+import React, { createContext, useState } from "react";
+import Navbar from "@/Components/Navbar/Navbar";
+import SensorBoxContainer from "@/Components/SensorBoxes/SensorBoxContainer";
+import ControlPanel from "@/Components/ControlPanel/ControlPanel";
+interface ConsoleContextProps {
+	console: number[];
+	setconsole: React.Dispatch<React.SetStateAction<number[]>>;
+}
 
-import "./App.css";
+export const consoleContext = createContext<ConsoleContextProps | undefined>(
+	undefined
+);
```

Comment:
As mentioned before, let's focus on the preliminary visual design and limit the scope otherwise. Please remove the context for now.